src/main/java/dao/impl/OrderDaoImpl.java

```diff
@@ -19,12 +19,14 @@
 public class OrderDaoImpl implements OrderDao {
 
     private static final Logger LOGGER = Logger.getLogger(OrderDaoImpl.class);
-    private static final String CONFIRM_ORDER = "UPDATE `order` SET confirmed=true WHERE user_id =?";
-    private static final String GET_ORDER_ID_BY_USER = "SELECT id FROM `order` WHERE user_id =? ORDER BY id DESC";
+    private static final String CONFIRM_ORDER = "UPDATE `order` SET confirmed=true " +
```

Comment:
```suggestion
    private static final String CONFIRM_ORDER = "UPDATE `order` SET confirmed = true " +
```
add space in all queries between words